local-interchaintest/src/main.rs

```diff
@@ -41,18 +43,25 @@ fn main() -> Result<(), Box<dyn Error>> {
         .with_transfer_channels(NEUTRON_CHAIN_NAME, JUNO_CHAIN_NAME)
         .build()?;
 
-    let mut uploader = test_ctx.build_tx_upload_contracts();
+    let current_dir = env::current_dir()?;
 
-    // TODO: uncomment to deploy polytone
-    // uploader
-    //     .with_key(ACC0_KEY)
-    //     .send_with_local_cache(POLYTONE_PATH, LOCAL_CODE_ID_CACHE_PATH_NEUTRON)
-    //     .unwrap();
+    // with test context set up, we can generate the .env file for the icq relayer
+    utils::generate_icq_relayer_config(
+        &test_ctx,
+        current_dir.clone(),
+        JUNO_CHAIN_NAME.to_string(),
+    )?;
 
-    let current_dir = env::current_dir()?;
+    // start the icq relayer. this runs in detached mode so we need
+    // to manually kill it before each run for now.
+    start_icq_relayer()?;
 
+    let mut uploader = test_ctx.build_tx_upload_contracts();
     let orbital_core_local_path = format!("{}/artifacts/orbital_core.wasm", current_dir.display());
 
+    info!("sleeping to allow icq relayer to start...");
+    std::thread::sleep(Duration::from_secs(30));
+
```

Comment:
Really hate seeing sleep lol, isn't there a way to know if the relayer is up?

Reply:
hmm, maybe we could query the logs of that docker container and sleep until a certain log comes up? but in general it's a separate docker image and it runs in detached mode so nothing really gets returned from there..